demo/demo.html

```diff
@@ -5,31 +5,71 @@
     <script src="../dist/components.js"></script>
     <script src="https://cdn.groupbycloud.com/sfx-all-plugins-bundle.js"></script>
     <!-- <script src="./temp/sfx-all-plugins-bundle.js"></script> -->
+    <style>
+      sfx-product-variant {
+        width: 40px;
+        height: 40px;
+        display: inline-block;
+        background-size: cover;
+        margin: 2px;
+        overflow: hidden;
+        cursor: pointer;
+      }
+
+      sfx-product-variant img {
+        width: auto;
+        height: auto;
+        min-width: 40px;
+        min-height: 40px;
+        max-width: 100%;
+        max-height: 100%;
+      }
+    </style>
   </head>
 
   <body>
     <sfx-search-box id="main-search" placeholder="search" clearbutton searchbutton></sfx-search-box>
     <sfx-sayt searchbox="main-search"></sfx-sayt>
 
-    <sfx-search-box id="secondary-search" group="secondary" placeholder="search" clearbutton searchbutton></sfx-search-box>
+    <sfx-search-box
+      id="secondary-search"
+      group="secondary"
+      placeholder="search"
+      clearbutton
+      searchbutton
+    ></sfx-search-box>
     <sfx-sayt searchbox="secondary-search" group="secondary"></sfx-sayt>
 
     <sfx-products></sfx-products>
 
     <script>
       function parseRecord(record) {
         const data = record.allMeta;
-        const firstVariant = data.visualVariants[0];
+        const visualVariants = data.visualVariants;
+        const firstVariant = visualVariants[0];
         const nonvisualVariants = firstVariant.nonvisualVariants;
         if (!nonvisualVariants[0]) throw new Error('No nonvisual variants');
 
         return {
           data,
           firstVariant,
           nonvisualVariants,
+          visualVariants,
         };
       }
 
+      function parseVariants(variants) {
```

Comment:
@ChantalDesrochers  @jmbtutor  Just thinking, do we want to be building out so much js and css in the html file? I understand that this is meant to approximate customizations done by the client but would it be cleaner to abstract it to a 'transformer.js' file and import that?

Reply:
I'm not too concerned about it because this is only for our use, it's only used here, and we're probably not going to change it that often. In my opinion, the demo file should be clean enough that we can work in it, but it doesn't have to be so clean that it's exemplary.